docs/user-guide/sdks-using.md

```diff
@@ -0,0 +1,68 @@
+# Using Zowe SDKs
+
+Learn about using the Zowe Client Software Development Kits (SDKs). The SDKs enable you to build client applications and scripts that interface with the mainframe.
+
+The SDKs include programamatic APIs, each of which performs a particular mainframe task. For example, one API package provides the ability to upload and download z/OS data sets. You can leverage that package to rapidly build a client application that interacts with data sets.
+
+The following SDKs are available.
+- Zowe Node Client SDK
+- Zowe Python Client SDK
+
+## Getting started
+
+To get started with the SDKs, navigate to [Zowe.org Downloads](https://www.zowe.org/download.html) and choose a programming language from the **Zowe Client SDKs** section. The package is downloaded to your computer.
+
+<!--TODO List the artifacts that the user can expect to find in the package after downloading (files, readmes, etc..) -->
+
+After you download an SDK, import the `ZoweSDK` class into your project. Create an object to handle requests to and from the mainframe.
```

Comment:
This line should be moved inside the Python example section.

For the Node SDK, there isn't a "ZoweSDK" class, and what you need to import depends on what you want to use (e.g., `import { Download } from "@zowe/zos-files-for-zowe-sdk"` or `import { Submit } from "@zowe/zos-jobs-for-zowe-sdk"`).

Reply:
Is there some step that we're missing, in between the user clicking "download" on Zowe.org and being able to use that` import { } from @zowe/zos-files-for-zowe-sdk` in their code? 

Do they need to copy the entire SDK folder to a certain directory in the project first? 

I wonder the same for Python.

Reply:
If users download the Zowe SDKs in a bundle from Zowe.org, then they would need to copy the package TGZs to their project where they want to install them so that they can run `npm install <packageName>.tgz`.

I expect that most SDK users will want to install packages directly from the NPM registry into their project, rather than pulling them from Zowe.org, so that they can declare the SDK packages as dependencies in their project's `package.json`.

This is different from the CLI situation, where the CLI and CLI plugins are globally installed and normally not added to a `package.json`. Because of this it doesn't matter whether CLI packages are installed from a local TGZ or the NPM registry.

Reply:
@tjohnsonBCM Thank you! this comment spurred me to add seperate sections for installing from package vs. registry. I reorganzied it a bit, it's worth another skim to make sure I explained it correctly. 

For installing from registry, I could use help with the exact steps. I left a comment. Appreciate the help.